husky_directory/templates/result_count.html

```diff
@@ -1,10 +1,22 @@
 <div class="rsummary" id="rsummary">
-    {% for scenario in search_result['scenarios'] %}
-        {% if (scenario['people']|length) %}
-            {# when done this needs to look something lik <a href="#f1">1 Faculty/Staff</a> #}
-            {{ scenario['description']}}: {{ scenario['people']|length }}
-            <br>
-        {% endif %}
-    {% endfor %}
+    {% if search_result['num_results'] > 0 %}
+        {% for scenario in search_result['scenarios'] %}
+            {% if scenario['num_results'] > 0 %}
+                {%  set description = scenario['description'] %}
+                {{ description }}:
+                {% for population, query_output in scenario['populations'].items() %}
+                    {% set result_count = query_output['num_results'] %}
+                    {% if result_count > 0 %}
+                        {# when done this needs to look something lik <a href="#f1">1 Faculty/Staff</a> #}
+                        {% set anchor_id = 'todo-coming-soon' %}
+                        <a href="#{{ anchor_id}}">
+                            {{ result_count }} {{ population|externalize }}
+                        </a>
+                        <br>
```

Comment:
I think I need to get rid of this `<br>`:

![Screen Shot 2021-03-29 at 3 57 43 PM](https://user-images.githubusercontent.com/1092941/112910097-8463c000-90a7-11eb-8566-5bd2b45a7f21.png)

should look like:

![Screen Shot 2021-03-29 at 3 58 54 PM](https://user-images.githubusercontent.com/1092941/112910170-ad845080-90a7-11eb-8aff-65600662e9d9.png)